utils/rewarder_configs.py

```diff
@@ -35,7 +35,7 @@ def formatRewarderConfigItem(rewarder_config_item):
     id = rewarder_config_item["PoolId"]
     pool = {
         "LP": Web3.toChecksumAddress(rewarder_config_item["LPToken"]),
-        "LPType": "",
+        "LPType": "StableAMM" if id in V2_STABLEPOOL_METADATA else "",
```

Comment:
If the poolID is in the `V2_STABLEPOOL_METADATA`, classify the farm as a stable pool farm.  This way, the existing gnosis flow doesn't need to change.

The only thing that would make this 100% automated was if the output at `https://raw.githubusercontent.com/trisolaris-labs/trisolaris_core/main/rewarderConfigs.json` could have a key/value pair of `LPType: "StableAMM" | ""`

cc @rej156